dkist/io/asdf/resources/schemas/tiled_dataset-0.1.0.yaml

```diff
@@ -17,7 +16,7 @@ properties:
     items:
       type: array
       items:
-        $ref: "http://dkist.nso.edu/schemas/dkist/dataset-0.3.0"
+        $ref: "asdf://dkist.nso.edu/schemas/dataset-0.3.0"
```

Comment:
Consider `tag:` validator here

Reply:
This one causes the validation with the top level schema to fail with 
```
E           jsonschema.exceptions.ValidationError: mismatched tags, wanted 'tag:dkist.nso.edu:dkist/dataset-0.3.0', got 'asdf://dkist.nso.edu/tags/dataset-1.0.0'
```
which I can not understand.

Reply:
I think that's because you're using the tiled_dataset-0.1.0 schema for both tags in the manifests:

```yaml
  - schema_uri: "asdf://dkist.nso.edu/schemas/tiled_dataset-0.1.0"
    tag_uri: "tag:dkist.nso.edu:dkist/tiled_dataset-0.1.0"
```
```yaml
  - schema_uri: "asdf://dkist.nso.edu/schemas/tiled_dataset-0.1.0"
    tag_uri: "asdf://dkist.nso.edu/tags/tiled_dataset-1.0.0"
```

Since that's the case, you could use a combiner here:

```yaml
      items:
        oneOf:
          - tag: tag:dkist.nso.edu:dkist/dataset-0.3.0
          - tag: asdf://dkist.nso.edu/tags/dataset-1.0.0
```

(or write separate schemas, but the I think the combiner is a nice solution)

Reply:
ah, I do actually have new schemas anyway, so thanks for spotting that!